src/Time/StepControllers/StepController.hpp

```diff
@@ -0,0 +1,50 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines base class StepController
+
+#pragma once
+
+#include "Time/Time.hpp"
+//#include "Options/Factory.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// Holds all the StepControllers
+namespace StepControllers {}
+
+namespace StepControllers {
+class BinaryFraction;
+class FullSlab;
+class SimpleTimes;
+class SplitRemaining;
+}
+
```

Comment:
Add to TimeSteppers doxygen group, or later create StepControllers doxygen group.